src/parser/jobs/sam/modules/ThirdEye.js

```diff
@@ -0,0 +1,86 @@
+import {ActionLink} from 'components/ui/DbLink'
+import React from 'react'
+import {Trans, i18nMark} from '@lingui/react'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+export default class ThirdEye extends Module {
+	static handle = 'Third Eye'
+	static i18n_id = i18nMark('sam.thirdeye.title')
```

Comment:
```suggestion
	static title = t('sam.thirdeye.title')`Third Eye`
```

(You'll need to `import {t} from '@lingui/macro'`)

Sorry, it's a super recent change! i18nMark still works, but I'm trying to remove it Soon:tm:.